doc-content/enterprise-only/processes/process-instance-details-con.adoc

```diff
@@ -1,7 +1,7 @@
 [id='process-instance-details-con-{context}']
 = Process instance management
 
-To view process instances, in Business Central, click *Menu* -> *Manage* -> *Process Instances*. Each row in the *Manage Process Instances* list represents a process instance from a particular process definition. Each execution is differentiated from all the others by the internal state of the information that the process is manipulating. Click a process instance to view the corresponding tabs with runtime information related to the process.
+To view process instances, in {CENTRAL} click *Menu* -> *Manage* -> *Process Instances*. Each row in the *Manage Process Instances* list represents a process instance from a particular process definition. Each execution is differentiated from all the others by the internal state of the information that the process is manipulating. Click a process instance to view the corresponding tabs with runtime information related to the process.
```

Comment:
I still don't like "each execution is differentiated from all the others" - what is "an execution"? Maybe "each instance"?